### DISCUSSION TITLE
lilive script link to groovy pdf: need of maintenance?

### DISCUSSION THREAD

**Discussion by Illegit** [freeplane/freeplane#2177] [Help] [answered]
[LinkToPdfPage2.zip](https://github.com/user-attachments/files/18269057/LinkToPdfPage2.zip)
Hi lilive, may you check your script? I'm using FP 1.12.8_3 and the script cannot any longer respect the page, the acrobat is communicationg an error and after opening just the pdf 1st page.
Is it possible to restore the correct behaviour?
Thank you.

**Comment by Illegit** [comment]
![image](https://github.com/user-attachments/assets/f19f80fd-1543-4cc3-97c8-f194850448de)
here is my acrobat release.

**Reply by lilive** [reply]
Hi @Illegit ,
I do not use this script anymore, but maybe I can help you to fix it.
The script you've post create a link that run SumatraPDF, not Acrobat. Are you sure this is the script you use ? Have you tried to follow the comments that explain how to set the script for Acrobat (that I don't use neither) ?

**Comment by Illegit** [comment]
[LinkToPdfPageFull.zip](https://github.com/user-attachments/files/18282781/LinkToPdfPageFull.zip)
sorry lilive I mixed the files: here the one w problem:
it creates the node with the link, but it opens the application Acrobat providing this warning
![png-241231-180222433-11009044940616816912](https://github.com/user-attachments/assets/c0d324c6-4131-487a-b93f-a50ffcc3f1c9)
then clicking OK
it opens the file at page 1
and not at the required page

**Comment by lilive** [comment, marked as answer]
> Have you tried to follow the comments that explain how to set the script for Acrobat ?

Comments recommend to replace lines 209 and 212 by 216 and 217. Your line 209 seems fine, but not the 212.

**Comment by Illegit** [comment]
Thank you, live. I forgot to add the args correctly because I was using Sumatra before. Now it is working!

**Reply by lilive** [reply]
OK

PS: If you use Windows, have you seen [this](https://github.com/freeplane/freeplane/discussions/2166#discussioncomment-11686279) ?

**Reply by Illegit** [reply]
Thank you lilive, it is interesting. I will go deeper and think how to use that. Nice tool.
